app/views/includes/upscan_file_upload_form.scala.html

```diff
@@ -0,0 +1,47 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.templates.spinner
+
+@(fileType: String, formFields: Map[String, String], postTarget: String)(implicit messages: Messages)
+
+<div id="uploadForm">
+  <form action="@postTarget" method="post" enctype="multipart/form-data">
+
+    @for(field <- formFields) {
+      <input type="hidden" name="@field._1" value="@field._2"/>
+    }
+
+    <div class="js-visible mt-3">
```

Comment:
can these div's be flattened if they are only for adding css classes?

Reply:
Done some testing and found this does not have any effect on the display. Amended